jetbrains-rider/src/software/aws/toolkits/jetbrains/services/clouddebug/DotNetDebuggerSupport.kt

```diff
@@ -494,44 +396,14 @@ class DotNetDebuggerSupport : DebuggerSupport() {
 
         val linuxSubdirectoryName = "linux-x64"
 
-        if (!useDotnetCoreRuntime) {
-            // Copy runtime.sh
-            val runtimeSh = RiderEnvironment.getBundledFile("runtime.sh")
-            FileUtil.copy(runtimeSh, File(targetPath, runtimeSh.name))
-
-            // Copy mono
-            val linuxMono = RiderEnvironment.getBundledFile(linuxSubdirectoryName, allowDir = true)
-            FileUtil.copyDir(linuxMono, File(targetPath, linuxMono.name))
-        } else {
-            val linuxMonoSubdirectory = File(targetPath, linuxSubdirectoryName)
-            if (linuxMonoSubdirectory.isDirectory) {
-                try {
-                    // remove existing linux Mono distribution since we run debugger on container .net core
-                    linuxMonoSubdirectory.deleteRecursively()
-                } catch (e: Throwable) {
-                    logger.trace(e) { "Error while trying to delete unused linux Mono directory ${linuxMonoSubdirectory.absolutePath}" }
-                }
+        val linuxMonoSubdirectory = File(targetPath, linuxSubdirectoryName)
+        if (linuxMonoSubdirectory.isDirectory) {
+            try {
+                // remove existing linux Mono distribution since we run debugger on container .net core
+                linuxMonoSubdirectory.deleteRecursively()
```

Comment:
assume this is to save space? is this destructive to the users local setup?

Reply:
It copies everything into our own directory and then removes mono

Reply:
Space is not the concern, it is the amount we have to rsync over. Copying mono slows down the copy by a few minutes if I remember right...

Reply:
This logic also has not hanged, it was the else branch of not using dotnet core. Using dotnet core debugger has been the default for us for 193+, but we had a registry option to disable in case of instability was found. This registry option has been removed since no issues have been found

Reply:
> it is the amount we have to rsync over.

this is what I meant